src/drivers/nuopc/cime_config/config_component_cesm.xml

```diff
@@ -201,7 +201,9 @@
       <value compset="_CAM\d+%WCXI">288</value>
       <value compset="_DATM%COPYALL_NPS">72</value>
       <value compset="_DATM.*_CLM">48</value>
-      <value compset="_DATM.*_DICE.*_POP2">4</value>
+      <!-- NOTE: if the following is changed from 24 -> 4 - then a new run sequence is needed since the -->
+      <!-- ocn coupling interval for this case is 24 - but the atm is longer -->
+      <value compset="_DATM.*_DICE.*_POP2">24</value>
```

Comment:
this is a change that would affect the mct comp_interface as well is it not?

Reply:
this will not effect the mct performance since this is in the nuopc/cime_config and NOT the mct/cime_config - they are totally separate